Bottom: 958bafca1e2193c227881d67d71a9f7bc3f358a8
Top:    b7a3d6f396d67561e2533fe6b22359de2e4bf088
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-04-25 15:24:09 -0500

Wait on openldap container startup

Occasionally the openldap container would not be ready when attempting
to add the admin and proxyagent accounts. A wait is required to
ensure the service is running properly.


---
diff --git a/roles/docker-openldap/tasks/main.yml b/roles/docker-openldap/tasks/main.yml
index e771b77..35c5ba7 100644
--- a/roles/docker-openldap/tasks/main.yml
+++ b/roles/docker-openldap/tasks/main.yml
@@ -149,6 +149,13 @@
 # *****************************************************************************
 # Set the LDAP passwords
 
+- name: wait for LDAP to start
+  wait_for:
+    delay: 5
+    host: '{{ container_addr_map.ldap.ip_addr }}'
+    port: 389
+    state: started
+
 - name: set LDAP admin password
   command: 'docker exec -i {{ openldap_container_name }} /docker-entrypoint.sh set_admin_pass {{ ldap_admin_password | quote }}'
